packages/massa-web3/src/experimental/account.ts

```diff
@@ -112,15 +114,15 @@ export class Account {
         }
         const passwordSeal = new PasswordSeal(password, salt, nonce)
         return {
-          address: this.address.toString(),
+          Address: this.address.toString(),
           Version: this.version,
-          nickname: '',
-          salt: passwordSeal.salt,
-          nonce: passwordSeal.nonce,
-          cipheredData: await passwordSeal
+          Nickname: '',
+          Salt: passwordSeal.salt,
+          Nonce: passwordSeal.nonce,
+          CipheredData: await passwordSeal
             .seal(this.privateKey.toBytes())
             .then((a) => Array.from(a)),
-          publicKey: Array.from(this.publicKey.toBytes()),
+          PublicKey: Array.from(this.publicKey.toBytes()),
         } as AccountV0KeyStore
```

Comment:
@peterjah I don't really know what is the difference between version 0 and version 1 of the wallet. Do you?